qemu/tests/cfg/migrate.cfg

```diff
@@ -198,6 +198,7 @@
             type = migration_with_file_transfer
         - with_netperf:
             no JeOS
+            only Linux
```

Comment:
Any reason? I see this variant has `Windows` support.

Reply:
Hi @PaulYuuu, I put the log in bug1883390. Because there is another script(netkvm_in_use.with_live_migration) that can cover this scenario, so I didn't fix it.

Reply:
If you want to remove windows support, you need to also remove windows related code from cfg and py files.

Reply:
Okay, I will make more reasonable changes in the next update, thanks.